editor-components/stats/stats8/stats8.tsx

```diff
@@ -3,112 +3,300 @@ import { BaseStats } from "../../EditorComponent";
+        "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/66a36aa42f8a5b002ce6a087?alt=media",
+    });
+
+    this.addProp({
+      type: "number",
+      key: "animationDuration",
+      displayer: "Number Animation Duration (ms)",
+      value: 500,
+    });
+
+    this.castToObject<ICard[]>("stats").map((statsData, index) =>
+      this.setComponentState(`number-${index}`, 0)
+    );
+
+    let interval = setInterval(() => {
+      this.castToObject<ICard[]>("stats").map(
+        (statData: ICard, index: number) => {
+          let currentNumber = this.getComponentState(`number-${index}`);
+          if (currentNumber != statData.title) {
+            this.setComponentState(
+              `number-${index}`,
+              Math.min(
+                statData.title,
+                currentNumber +
+                  Math.ceil(
+                    statData.title /
+                      Math.round(this.getPropValue("animationDuration") / 30)
+                  )
+              ) || 0
+            );
+          }
+        }
+      );
+    }, 30);
   }
 
   getName(): string {
     return "Stats 8";
   }
 
   render() {
+    const statsData = this.castToObject<ICard[]>("stats");
+    const imageSrc = this.getPropValue("imageSrc");
+
+    const title = this.castToString(this.getPropValue("title"));
+    const subtitle = this.castToString(this.getPropValue("subtitle"));
+    const description = this.castToString(this.getPropValue("description"));
+    const author = this.castToString(this.getPropValue("author"));
+    const authorRole = this.castToString(this.getPropValue("authorRole"));
+    const overlayNumber = this.getPropValue("overlayNumber");
+    const overlayDescription = this.castToString(
+      this.getPropValue("overlayDescription")
+    );
+
+    const isContentPresent =
+      title ||
+      subtitle ||
+      description ||
+      author ||
+      authorRole ||
+      statsData.length > 0;
+
     return (
       <div className={this.decorateCSS("container")}>
-        <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("stats2-page")}>
-            <div className={this.decorateCSS("bottom-child")}>
-              <div className={this.decorateCSS("card-title")}>
-                <h1 className={this.decorateCSS("title")}>{this.getPropValue("title")}</h1>
+        <div
+          className={
+            this.decorateCSS("max-content") +
+            (!isContentPresent ? " full-width" : "")
+          }
+        >
+          {isContentPresent && (
+            <div className={this.decorateCSS("stats2-page")}>
+              <div className={this.decorateCSS("content")}>
+                {title && (
+                  <h2
+                    className={
+                      this.decorateCSS("title") +
+                      " " +
+                      this.decorateCSS("text-uppercase")
+                    }
+                  >
+                    {title}
```

Comment:
use this.getPropValue cause you are using cast to string above. if you are using string the inline editor does not appear in strings